### DISCUSSION TITLE
Can I configure PrivateGPT with Azure OpenAI keys? How to configure? please help.

### DISCUSSION THREAD

**Discussion by mraguth** [imartinez/privateGPT#1204] [Q&A]
I like to use my own Azure OpenAI keys for this application because I restricted to use only company-preferred AI in my company.

**Comment by rostrovsky** [comment]
I'm interested in that feature too. Is this currently possible or would require PR?

**Comment by pkandarpa-cs** [comment]
Check here - https://github.com/zylon-ai/private-gpt/blob/347be643f7929c56382a77c3f45f0867605e0e0a/private_gpt/settings/settings.py#L258

and there's a settings file called settings-azopenai.yaml in the latest version with details
